views/auth/claim/pages/health-card2.ejs

```diff
@@ -43,7 +43,7 @@
 						</div>
 
 						<div class="bank-details-container" style="display:none;">
-							<div class="col-xs-12 col-sm-8 col-lg-6" style="border:1px solid #cccccc; padding:2em; margin:0 0 2em 0;">
+							<div class="col-xs-12 col-sm-8 col-lg-6 boxed">
 								<button id="btnCancelBank" class="uikit-btn uikit-btn--tertiary no-left-margin small floated " style="display: none">Cancel</button>
 								<p style="padding-top:0.4em;">All fields are required.</p>
```

Comment:
Still a couple of inline styles here - note that these won't translate across to dev unless devs make changes to the .sass files for impacted pages/components in our repo.  It can be done in specific circumstances, but it's not a good long term practice as it'll separate the look and feel into two places.

Feel free to ignore me if these styles aren't necessary in dev.